acquia_lift_profiles/acquia_lift_profiles.admin.inc

```diff
@@ -54,16 +54,25 @@ function acquia_lift_profiles_admin_form($form, &$form_state) {
     '#description' => t('Check this if you want Acquia Lift Profiles to capture the identity of the user upon login or registration. This means sending their email address to the Acquia Lift Profile Manager.'),
     '#default_value' => variable_get('acquia_lift_profiles_capture_identity', FALSE),
   );
+  $identity_parameter_display_value = variable_get('acquia_lift_profiles_identity_param') ?: 'identity';
+  $identity_type_parameter_display_value = variable_get('acquia_lift_profiles_identity_type_param') ?: 'identityType';
+  $default_identity_type_display_value = variable_get('acquia_lift_profiles_default_identity_type') ?: ACQUIA_LIFT_PROFILES_DEFAULT_IDENTITY_TYPE_DEFAULT;
```

Comment:
Ooh, for some reason I thought the ?: shorthand required a later version of PHP but I just checked and it works in 5.3 so we're good :)